### DISCUSSION TITLE
finding difficulties while restoring my website

### DISCUSSION THREAD

**Discussion by 2602shubham** [elementor/elementor#23525] [Developers Community]
I'm finding difficulties while restoring my website. Please help me out.
![2023-08-23](https://github.com/elementor/elementor/assets/142936446/e0f5896d-921c-45b3-b3f5-bd4c5f0ac206)
At 61% it get stuck and lastly its getting failed.

**Comment by toineenzo** [comment]
This has nothing to do with Elementor. Try contacting the Backuply developers
